src/main/java/com/sinch/xms/InboundsFilter.java

```diff
@@ -85,6 +85,15 @@ public int pageSize() {
    */
   public abstract Set<String> recipients();
 
+  /**
+   * Limits results to inbound messages sent with the specified client reference. If empty then all
+   * client references are eligible.
+   *
+   * @return a client reference
+   */
+  @Nullable
+  public abstract String clientReference();
```

Comment:
Client reference in MOs has not yet been revealed, need to check with PM if this should be here now or hold until later